apps/native/app/src/screens/family/family-details.tsx

```diff
@@ -93,45 +139,56 @@ export const FamilyDetailScreen: NavigationFunctionComponent<{
               loading={loading}
               error={!!error}
             />
-            {person?.nationality ? (
+            {'citizenship' in person && person?.citizenship ? (
               <Input
                 label={intl.formatMessage({
                   id: 'familyDetail.natreg.citizenship',
                 })}
-                value={person?.nationality}
+                value={person?.citizenship?.name}
                 loading={loading}
                 error={!!error}
               />
             ) : null}
           </InputRow>
-          {person?.legalResidence ? (
+          {'housing' in person &&
+          person?.housing &&
+          'address' in person.housing &&
+          person.housing.address ? (
             <InputRow>
               <Input
                 label={intl.formatMessage({
                   id: 'familyDetail.natreg.legalResidence',
                 })}
-                value={person?.legalResidence}
+                value={
+                  'postalCode' in person.housing.address &&
```

Comment:
personally I prefer using optional chaining instead of the in syntax
Would be nice to store in a var above to have shorter jsx
```
const address = person?.housing.address
```

Reply:
I agree, I prefer that too. But it is not enough in this case, I get a ts error if I only do normal optional chaining.